packages/node-cli/src/commands/object/update/UpdateAction.js

```diff
@@ -47,26 +53,85 @@ module.exports = class UpdateAction extends BaseAction {
 			if (params.hasOwnProperty(ANSWERS_NAMES.OVERWRITE_OBJECTS) && !params[ANSWERS_NAMES.OVERWRITE_OBJECTS]) {
 				throw NodeTranslationService.getMessage(MESSAGES.CANCEL_UPDATE);
 			}
-			const sdkParams = CommandUtils.extractCommandOptions(params, this._commandMetadata);
-
-			const executionContextForUpdate = SdkExecutionContext.Builder.forCommand(this._commandMetadata.sdkCommand)
-				.integration()
-				.addParams(sdkParams)
-				.build();
-
-			const operationResult = await executeWithSpinner({
-				action: this._sdkExecutor.execute(executionContextForUpdate),
-				message: NodeTranslationService.getMessage(MESSAGES.UPDATING_OBJECTS),
-			});
-
-			return operationResult.status === SdkOperationResultUtils.STATUS.SUCCESS
-				? ActionResult.Builder.withData(operationResult.data)
-					.withResultMessage(operationResult.resultMessage)
-					.withCommandParameters(sdkParams)
-					.build()
-				: ActionResult.Builder.withErrors(operationResult.errorMessages).withCommandParameters(sdkParams).build();
+			let totalData = [];
+			if (params.hasOwnProperty(COMMAND_OPTIONS.INCLUDE_INSTANCES) && params[COMMAND_OPTIONS.INCLUDE_INSTANCES]) {
+				const customRecordScriptIds = params[COMMAND_OPTIONS.SCRIPT_ID]
+					.split(' ')
+					.filter((scriptId) => this._isCustomRecordOrSegment(scriptId));
+				params[COMMAND_OPTIONS.SCRIPT_ID] = params[COMMAND_OPTIONS.SCRIPT_ID]
+					.split(' ')
+					.filter((scriptId) => !this._isCustomRecordOrSegment(scriptId))
+					.join(' ');
+				const updateCustomRecordWithInstancesResult = await this._updateCustomRecordWithInstances(params, customRecordScriptIds);
+				totalData = totalData.concat(updateCustomRecordWithInstancesResult);
+
+				delete params[COMMAND_OPTIONS.INCLUDE_INSTANCES];
+			}
+
+			if (params[COMMAND_OPTIONS.SCRIPT_ID] === '') {
+				// If there are no more objects to update (all were custom records) we already return with only previous messages
+				return ActionResult.Builder.withData(totalData).build();
+			}
+
+			return await this._updateObjects(params, totalData);
```

Comment:
Not sure if we should be returning the result of the "normal" `this._updateObjects` and merge the results here to have more clarity on what we are doing. In this way we would not need to pass the "totalData" to `this._updateObjects`